### PR TITLE
Upgrade Docker image to Alpine 3.11

### PR DESCRIPTION
**Pull request by languitar**
This make tesseract 4.1 avaialbe, which fixes some things like infinite
processing loops on some documents:
https://github.com/tesseract-ocr/tesseract/issues/2288#issuecomment-561727361

Some dependencies had to be bumped for being compatible with the new Alpine
libraries.

### CODE REVIEW

**Comment by languitar** [comment]
Travis failure seem to be unrelated to my changes. They originate from the sphinx documentation :/

File: src/paperless/settings.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def __get_boolean(key, default="NO"):
 STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
 
 # We allow CORS from localhost:8080
-CORS_ORIGIN_WHITELIST = tuple(os.getenv("PAPERLESS_CORS_ALLOWED_HOSTS", "localhost:8080").split(","))
+CORS_ORIGIN_WHITELIST = tuple(os.getenv("PAPERLESS_CORS_ALLOWED_HOSTS", "http://localhost:8080").split(","))
```
Comment (bauerj) [inline src/paperless/settings.py]: How is this related to upgrading the operating system?

Comment (languitar) [reply src/paperless/settings.py]: Sorry, forgot to mention this in the commit message. After upgrading the packages in the lock file, some upgraded dependency for cors now requires full URIs instead of hostnames now.

Comment (bauerj) [reply src/paperless/settings.py]: Okay, thanks for clarifying :+1:

**Comment by Tooa** [comment]
> Travis failure seem to be unrelated to my changes. They originate from the sphinx documentation :/

@languitar @bauerj Merge PR https://github.com/the-paperless-project/paperless/pull/601 first to fix this issue

**Comment by pitkley** [comment]
@languitar thanks for the PR, dependency updates are always nice! #601 is just missing another review from @the-paperless-project/reviewers, then you can rebase on master to fix the Travis issue.

Regarding the `setrlimit(RLIMIT_CORE)` error/warning @Tooa mentioned: a simple fix is to `echo 'Set disable_coredump false' >> /etc/sudo.conf`, see https://ask.fedoraproject.org/t/sudo-setrlimit-rlimit-core-operation-not-permitted/4223 and https://gitlab.alpinelinux.org/alpine/aports/issues/11122. Maybe the suggested `doas`-change in the latter link is a more correct fix, but I think that would be too big for what this PR should be doing.

Feel free to ping me directly for a review once the Travis build is fixed. 👍

**Comment by languitar** [comment]
I have rebased the PR, added the setrlimit fix, and split the PR into two distinct commits with further clarifications.

File: src/paperless/settings.py (hunk 1 above)
Comment (MasterofJOKers) [reply src/paperless/settings.py]: Can we mention this setting in [the docs](https://github.com/the-paperless-project/paperless/blame/master/docs/setup.rst#L196)? I would expect people having SSL enabled to change this. Or can we get away with `//` instead of `http://` to support both?

Comment (languitar) [reply src/paperless/settings.py]: I think a proper URL is required now. So // probably doesn't work.

Comment (bauerj) [reply src/paperless/settings.py]: You could just add both URLs, couldn't you?

Comment (languitar) [reply src/paperless/settings.py]: done
